packages/wonder-blocks-breadcrumbs/components/breadcrumbs.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import type {AriaProps} from "@khanacademy/wonder-blocks-core";
+import {addStyle} from "@khanacademy/wonder-blocks-core";
+import BreadcrumbsItem from "./breadcrumbs-item.js";
+
+type Props = {|
+    ...AriaProps,
+
+    /**
+     * This is the content for the collection of Breadcrumbs
+     *
+     */
+    children: React.ChildrenArray<React.Element<typeof BreadcrumbsItem>>,
+
+    /**
+     * Test ID used for e2e testing.
+     */
+    testId?: string,
+|};
+
+const StyledList = addStyle("ol");
+
+/**
+ * A breadcrumb trail consists of a list of links to the parent pages
+ * of the current page in hierarchical order. It helps users find their
+ * place within a website or web application.
+ * Breadcrumbs are often placed horizontally before a page's main content.
+ *
+ * The Breadcrumbs component will have the following structure:
+ *
+ * 1. Breadcrumbs Item: Represents a section within the page.
+ * 2. Separator: Adds a separator between each item.
+ */
+export default class Breadcrumbs extends React.Component<Props> {
+    render() {
+        const {children} = this.props;
+        // using React.Children allows to deal with opaque data structures
+        // e.g. children = 'string' vs children = []
+        const numChildren = React.Children.count(children) - 1;
```

Comment:
This should probably be renamed to `numSeparators` or `lastChildIndex`.

Reply:
👍  I'll go with `lastChildIndex` (b/c of the variable intention)